app/bundles/PluginBundle/Integration/AbstractIntegration.php

```diff
@@ -2704,4 +2704,18 @@ public function getCompoundMauticFields($lead)
 
         return $lead;
     }
+
```

Comment:
Please add some annotation of the method so we'll know what it is for in the far future. It would be also easy to write a unit test for it.